_ml-commons-plugin/ml-framework.md

```diff
@@ -48,7 +48,7 @@ To achieve better performance within the ML Framework, you can take advantage of
 Use the URL upload operation for models that already exist on another server, such as GitHub or S3. 
 
 ```
-POST /_plugins/_ml/models/_upload
+POST /_plugins/_ml/models/_register
```

Comment:
Can we please copy [this request field](https://opensearch.org/docs/latest/ml-commons-plugin/api/#request-fields) table here?